### ISSUE TITLE
Want to restrict the height of the wave

### ISSUE THREAD

**Issue by anaskhanstack** [katspaugh/wavesurfer.js#2266]
In documentation there is written that Height of the waveform bars. Higher number than 1 will increase the waveform bar heights.
Is there any possibility to restrict the wavebar till one or lower (0.07)?

Any help would be aprreciated since I can't find a solution by myself

![image](https://user-images.githubusercontent.com/45259204/117571262-3bac1780-b0e7-11eb-939e-8fb1a2d09728.png)

Version: "wavesurfer.js": "^4.6.0"

**Comment by thijstriemstra** [comment]
what have you tried? Try the `barHeight` option.

**Comment by anaskhanstack** [comment]
I have tried the barHeight but it does not restrict the wavebar till 1

I tried to do one thing

  const optimizeWavebar = () =>{
    var length = 302;
    var start = 0;
    var end = 302;
    let surfer = wavesurfer.current.backend.getPeaks(length, start, end)
    return surfer.map(index=> index >= 0.05 ? 0.05 : index <= -0.05 ? -0.05 : index)
  }
  
   useEffect(() => {
    const options = formWaveSurferOptions(waveformRef.current);
    wavesurfer.current =  WaveSurfer.create(options);
    wavesurfer.current.load(track.url);

    wavesurfer.current.on("ready", () => {   
       wavesurfer.current.load(track.url,optimizeWavebar());   
      if (wavesurfer.current) {
        wavesurfer.current.setVolume(0);
        setDuration(format(wavesurfer.current.getDuration()));
      }
    });

    return () => wavesurfer.current.destroy();
  }, [track.url]);

Basically what i have done is that i load the waves at first and then get the peaks of wave and refine it that if bar crosses 1 i should re assign to 1 again i.e. optimizeWavebar() but it does not work well for me.

These are the things that i have tried so far

**Comment by sundayz** [comment]
We have `barMinHeight` maybe we should have `barMaxHeight` too?

You can try generating your own peaks on the server, that way you have full control over what they look like.

**Comment by sundayz** [comment]
Also try using the `normalize` option and see what effect that has: https://wavesurfer-js.org/docs/options.html
